### ISSUE TITLE
Webstorm's "unresolve function or method" using this styleguide

### ISSUE THREAD

**Issue by admon-s-waklme** [johnpapa/angular-styleguide#659]
Hi,

When using the following code, the "getLang" function isn't recognized by WebStorm 11.0
`    
angular.module('app').controller('languageCtrl', languageCtrl);
languageCtrl.$inject = [];
function languageCtrl() {
	var vm = this;
	vm.getLang = getLang;
	function getLang() {}
}	
`
When changing the function name to something else than the controller name (e.g. languageController), the controller method are recognized.

Any idea how to fix this?

**Comment by cyberkatze** [comment]
What do you mean exactly?

In my case on 11.0.3 work without problem:

```js
angular.module('app').controller('languageCtrl', languageCtrl2);

languageCtrl2.$inject = [];

function languageCtrl2() {
        var vm = this;
        vm.getLang = getLang;
        function getLang() {
        }
}
````

**Comment by admon-s-waklme** [comment]
Yes, because you're using "languageCtrl**2**", that's exactly my problem.
If you'll use "languageCtrl" it won't work

**Comment by cyberkatze** [comment]
@admon-s-waklme :
Sorry, but I don't understand your problem.

If I use "languageCtrl"-controller instead of "languageCtrl**2**", it work on me without problem too.

![bildschirmfoto 2016-02-14 um 09 05 43](https://cloud.githubusercontent.com/assets/2271337/13032634/3557ec7c-d2fa-11e5-9e45-14ef6b4e8455.png)

**Comment by MAustinMMDP** [comment]
The function should be added to the body of the activate function or you can change the name of the activate function to getLang if you like. Regardless it works for me too. You may want to check your routing.

```javascript
(function () {
    'use strict';

    angular
        .module('app')
        .controller('languageCtrl', languageCtrl);

    languageCtrl.$inject = [''];

    /* @ngInject */
    function languageCtrl() {
        var vm = this;
        vm.title = 'languageCtrl';

        getLang();

        ////////////////

        function getLang() {
            
        }
    }

})();
```

**Comment by johnpapa** [comment]
is this still an issue?

**Comment by johnpapa** [comment]
Thanks for reaching out! This is a support question rather than a style guide issue. Stack Overflow and the Angular community are great resources for questions like this. Closing to keep the tracker focused. 🙏
